### DISCUSSION TITLE
Does logging to InfluxDB require Red Pitaya to have access to the internet?

### DISCUSSION THREAD

**Discussion by yasio90** [linien-org/linien#380] [Q&A] [answered]
Our institute experiences lately quite frequent network issues. To be independent of them, we consider having a LAN in our lab that would be independent of the institute network, but this would mean that Red Pitayas would have no access to the internet. However, computers controlling the Red Pitayas would still have the access to the internet. In such a configuration, would the logging to InfulxDB provided by Linien work?

**Comment by bleykauf** [comment, marked as answer]
This very much depends on where your InfluxDB is hosted. The important thing is that you still have access to the the HTTP API via the port you configured (default 8086), see https://docs.influxdata.com/influxdb/v2/install/?t=Kubernetes#networking-ports
